src/js/components/List/__tests__/List-test.tsx

```diff
@@ -454,7 +454,7 @@ describe('List events', () => {
   test('should apply pagination styling', () => {
     const { container } = render(
       <Grommet>
-        <List data={data} paginate={{ background: 'red', margin: 'large' }} />
+        <List data={data} paginate={{ margin: 'large' }} />
```

Comment:
It looks like `background` isn't actually a valid paginate prop

Reply:
Good catch, I think removing the background here is fine